### PR TITLE
Create Reading the mind in the eyes.md

### PR DESCRIPTION
**Pull request by sarikasu**

### CODE REVIEW

File: tasks/Reading the mind in the eyes.md [hunk 1]
```diff
@@ -0,0 +1,78 @@
...
+
+
+# Procedure
+## Steps
+1. Participants read consent form and agree to participate
+2. Participants are given written instructions of the task 
+3. Participants answer a practice/example problem. 
+4. Participants then complete 36 RME picture problems.  
+> In the RME test, participants are shown 36 pairs of eyes, where for each pair of eyes they had to choose one of four words describing the corresponding emotion 
+5. Participants then complete a short survey
```
Comment (DodgeHill) [inline tasks/Reading the mind in the eyes.md:57]: Do we have any information on questions asked in the survey?  I did not se anything in the material.

Comment (DodgeHill) [reply tasks/Reading the mind in the eyes.md:57]: Also there is no mention of the original test, I looked through the article referenced, but I think the information you have provided is solid.

**Review by DodgeHill — COMMENTED** [review]
Overall good, just the small point about the contents of the survey the participants answered.

**Review by sarikasu — COMMENTED** [review]
Added survey questions

**Review by DodgeHill — APPROVED** [review]
Looks good to me

**Pull request merged**